system/lang/pl.json

```diff
@@ -10,26 +10,48 @@
     "CustomTruth": "Własna Prawda",
     "Roll": "Rzuć",
     "Chat": "Czat",
-    "Rolling": "Losuje..",
+    "SendToChat": "Wyślij tekst na czat: <i>{move}</i>",
+    "CopyTextToClipboard": "Kopiowanie tekstu do schowka",
+   "Rolling": "Losuje..",
     "Bonus": "Bonus",
+    "Adds": "Dodaje",
     "Versus": "vs.",
-    "CappedAt10": "Capped at 10",
+    "CappedAt10": "Ograniczenie do 10",
     "Strong_hit": "Sukces!",
     "Strong_hit_match": "Triumf! (match)",
     "On_a_strong_hit": "Przy Pełnym Sukcesie",
     "Weak_hit": "Częściowy Sukces",
     "On_a_weak_hit": "Przy Częściowym Sukcesie",
-    "Miss": "Porazka",
+    "Miss": "Porażka",
     "Miss_match": "Krytyczna Porazka(match)",
     "On_a_miss": "Przy miss",
     "Opportunity": "Okazja!",
     "Complication": "Komplikacja!",
     "MarkProgress": "Zaznacz Postęp",
+    "ProgressMove": "Zmień Postęp",
+    "PROGRESS": {
+      "Current": "{score} postęp ({ticks} ticks)",
+      "Ticks": "{ticks} ticków",
+      "Track": "tor postępu",
+      "BOX": {
+        "1": "1 pole",
+        "2": "2 pola",
+        "3": "3 pola"
+      },
+      "TICK": {
+        "1": "1 tiki",
+        "2": "2 tiki",
+        "3": "3 tiki",
+        "4": "4 tiki"
+      }
+    },
+    "MarkLegacy": "Zaznacz {amount} na twoim {legacy} torze dziedzictwa",
     "UnmarkProgress": "Odznacz Postęp",
     "ClearProgress": "Wyczyść Postęp",
     "FulfillVow": "Wypełnij Przysięge",
     "ProgressRoll": "Rzut postępu",
-    "StarProgress": "Gdy jest aktywny, ten element pojawi się również w zakładce 'Dziedzictwo'",
+    "MakeAProgressRoll": "Wykonuj rzut postepu {score} postępu(????)",
```

Comment:
is this unfinished?

Reply:
No I will delete em, probably I left em for reminder bcuz i wasn't sure which translation would be best ;)